addons/website/static/src/js/menu/navbar.js

```diff
@@ -37,6 +38,12 @@ var WebsiteNavbar = publicWidget.RootWidget.extend({
      */
     start: function () {
         var self = this;
+            dom.initAutoMoreMenu(this.$('ul.o_menu_sections'), {
+                maxWidth : function () {
+                    // NOTE : navbar will contain different elements in community and enterprise, so we check for both of them here only
+                    return self.$el.width() - (self.$('.o_menu_systray').outerWidth(true) + (self.$('ul#oe_applications').outerWidth(true) || 0) + (self.$('.o_menu_toggle').outerWidth(true) || 0) + (self.$('.o_menu_brand').outerWidth(true) || 0));
+                }}
+            );
```

Comment:
Close the object on the last line and a trailing comma for each key-value:
```javascript
        }}
    );
```
=>
```javascript
        },
    });
```